src/xma/src/xmaapi/xma_utils.cpp

```diff
@@ -497,6 +547,12 @@ int32_t get_cu_index(int32_t dev_index, char* cu_name1) {
     return -1;
 }
 
+/// <summary>
+/// get_default_ddr_index() - Returns default ddr index for the given cu index.
```

Comment:
@stsoe curious tool where you need to repeat the obvious, the function name.

Reply:
Yeah, it is not needed.  My preference on commenting is to use just '//' lines before each function to explain what it does unless it is obvious.  The commenting in compilation units (.cpp files) is not used for anything (doxygen or the like) other than help the next developer.  I don't care enough about the exact style used.

Reply:
addressed this comment in the latest changes.